Tasks/AzureFunctionAppV1/deploymentProvider/BuiltInLinuxWebAppDeploymentProvider.ts

```diff
@@ -14,7 +14,18 @@ const premiumPlanRunsFromPackage: string = ' -WEBSITE_RUN_FROM_PACKAGE true';
 const linuxFunctionRuntimeSettingValue = new Map([
     [ 'DOCKER|microsoft/azure-functions-dotnet-core2.0:2.0', 'dotnet ' ],
```

Comment:
Can you remove values that start with `Docker|` ?

Reply:
We don't want to break/effect the existing pipelines so we are leaving this as is but these will not show in the dropdown. Values mentioned in task.json is shown to users in dropdown.

Reply:
For existing pipelines that use `Docker|` is there any way to show a warning to prompt the user to move to new format?  This is  related auto-upgrade issue: https://github.com/Azure/app-service-announcements/issues/266 , apps that are using `Docker|` cannot be auto-upgraded

Reply:
Within tasks framework we don't have any such option which we could use to warn our users. We can only add warning message in the logs. I'll do cross check with team once to make sure.

Reply:
Thanks! That helps. Please add link to above announcement as a warning in the logs for pipelines using old format `DOCKER|microsoft/azure-functions-*`

Reply:
We won't be able to add warning as a client. We will add an additional message in the info hover that values such as `DOCKER|microsoft/azure-functions-*` are deprecated and to use the new values from dropdown.
Ideally the service call to deploy should either fail or return the message which we will display to the users.